.github/workflows/release.yml

```diff
@@ -90,5 +104,18 @@ jobs:
         if: ${{ inputs.box_upload == 'true' }}
         run: |
           echo "Do something"
```

Comment:
I know you haven't done this but since you're changing this file, you can remove this pointless echo.

Reply:
done, deleted